jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -104,11 +107,34 @@ private static void assertExtractionPathsUnordered(
   @Mock private AuthProperty authProperty;
   @Mock private Consumer<LogEvent> logger;
 
+  /** Configuration for {@code platform} parameter. */
+  public static class PlatformParametersTest implements PlatformConfiguration {
```

Comment:
No need for a comment. And this class isn't going to be used outside of this test class, you don't want to make it `public`. And the name `-Test` doesn't sound right.

```suggestion
  private static class TestPlatformConfiguration implements PlatformConfiguration {
```